nix/sources.json

```diff
@@ -59,6 +59,18 @@
         "url": "https://github.com/nmattia/niv/archive/efce82e4ba77a84835f9febefe3b2fe52a8c7ede.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
+    "nix-linter": {
```

Comment:
Any reason not to use it from nixpkgs instead?

Reply:
I didn't find it in nixpkgs on initial implementation.
I can update PR.